### PR TITLE
Updating nemo example app to have a spec using nemo-page.

### PR DESCRIPTION
**Pull request by OuranosSkia**

### CODE REVIEW

**Comment by OuranosSkia** [comment]
Can someone review/pull this please!

File: test/functional/spec/page-spec.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+var util = require('../util');
+
+describe('@page@', function () {
+  before(function (done) {
+    nemo = Nemo(done);
+  });
+  after(function (done) {
+    nemo.driver.quit().then(done);
+  });
+  it('should execute high level functionality using generic methods', function (done) {
```
Comment (grawk) [inline test/functional/spec/page-spec.js]: Does this test name make sense? Are there non-generic methods? If so, there should be an example with those as well.

**Comment by grawk** [comment]
Either as part of this PR, or in the very near future, it'd be awesome to add a section to README about the `nemo-page` view abstraction with a link to the `nemo-page` repo.

**Comment by grawk** [comment]
Aside from the one question about the testcase name, and whether there should be an additional testcase to illustrate further functionality, this looks great!

**Comment by OuranosSkia** [comment]
Updated based on comments! Let me know if there is anything else.

**Comment by grawk** [comment]
:+1: it looks good to me.

**Pull request merged**